locales/tr/json.json

```diff
@@ -6,30 +6,30 @@
     "90 Days": "90 gün",
     ":amount selected": ":Amount seçildi",
     ":amount Total": "Toplam :amount",
-    ":days day trial": ":Days günlük deneme",
-    ":resource Details": ":Resource detayları",
-    ":resource Details: :title": ":Resource detayları: :title",
-    "A decryption key is required.": "A decryption key is required.",
+    ":days day trial": ":days günlük deneme",
+    ":resource Details": ":resource detayları",
```

Comment:
I think in Turkish, sentences start with a capital letter. See how Laravel handles string conversion:

https://github.com/orgs/Laravel-Lang/discussions/7

Reply:
I fixed on https://github.com/Laravel-Lang/lang/pull/5540